translator/translate/logs/logs_collected/files/collect_list/ruleLogFilters_test.go

```diff
@@ -62,3 +62,19 @@ func TestApplyLogFiltersRuleMissingConfigInsideFilters(t *testing.T) {
 	assert.Nil(t, retVal)
 	assert.Len(t, translator.ErrorMessages, 2)
 }
+
+func TestApplyLogFiltersRuleInvalidRegex(t *testing.T) {
+	translator.ResetMessages()
+	r := new(LogFilter)
+	var input interface{}
+	e := json.Unmarshal([]byte(`{
+		"filters": [
+			{"type": "exclude", "expression": "(?!re)"}
+		]
+	}`), &input)
+	assert.Nil(t, e)
+	retKey, retVal := r.ApplyRule(input)
+	assert.Equal(t, "filters", retKey)
```

Comment:
I don't think we need to assert retKey to begin with across the test since the final goal was to validate the errorMessage if having invalid reg_exp and the expect failed value